src/register/Register.jsx

```diff
@@ -80,15 +81,16 @@ export default function Register() {
                                     required
                                 />
                             </p>
-                            {!isValidName && (
-                                <span className="error">Must be between 2-30 characters long. Can only contain alphabets, whitespace and dash '-'</span>
-                            )}
+                         <div>{!isValidName && (
+                                <span className="error" title="errorname" aria-label="errorname" role="alert">Must be between 2-30 characters long. Can only contain alphabets, whitespace and dash '-'</span>
```

Comment:
please use camel-case here (and everywhere else), so "errorName" instead of "errorname"